bigchaindb/backend/exceptions.py

```diff
@@ -17,5 +17,9 @@ class DuplicateKeyError(OperationError):
     """Exception raised when an insert fails because the key is not unique"""
 
 
-class BigchainDBCritical(Exception):
-    """Unhandleable error that requires attention"""
+class CriticalDoubleSpend(BigchainDBError):
```

Comment:
AFAIK `CriticalDoubleSpend` is used in `core.py`, so I wonder why this exception is defined in `bigchaindb.backend.exceptions` and not in `bigchaindb.exceptions`.